scripts/addr2line.py

```diff
@@ -118,6 +118,9 @@ def _output(self):
     def _read_resolved_address(self):
         first = self._output.readline()
         self._parent.debug('Addr2Line read output (first): ', first)
+        # check for invalid address at first line, which might ends in `:0` not in `: 0`
+        if Addr2Line.dummy_pattern.fullmatch(first):
+            return "?? ??:0"
```

Comment:
I have no idea what this is about.
Could you add some example output in the comment, to illustrate what case is this guarding against?